test/widgets/autocomplete_test.dart

```diff
@@ -177,6 +177,67 @@ void main() {
 
       debugNetworkImageHttpClientProvider = null;
     });
+
+    testWidgets('user options appear in the correct rendering order and do not scroll down', (tester) async {
+      final List<User> users = [
+        eg.user(userId: 1, fullName: 'Aaditya', avatarUrl: 'user1.png'),
+        eg.user(userId: 2, fullName: 'Alya', avatarUrl: 'user2.png'),
+        eg.user(userId: 3, fullName: 'Aman', avatarUrl: 'user3.png'),
+        eg.user(userId: 4, fullName: 'Anders', avatarUrl: 'user4.png'),
+        eg.user(userId: 5, fullName: 'Anthony', avatarUrl: 'user5.png'),
+        eg.user(userId: 6, fullName: 'Apoorva', avatarUrl: 'user6.png'),
+        eg.user(userId: 7, fullName: 'Asif', avatarUrl: 'user7.png'),
+        eg.user(userId: 8, fullName: 'Asim', avatarUrl: 'user8.png')
+      ];
+
+      final composeInputFinder = await setupToComposeInput(tester, users:users);
+      final store = await testBinding.globalStore.perAccount(eg.selfAccount.id);
+
+      final expectedUserSequence = users;
+      // Options are filtered correctly for query
+      // TODO(#226): Remove this extra edit when this bug is fixed.
```

Comment:
In https://github.com/zulip/zulip-flutter/pull/1131/commits/b35ddbf51f9efacbde2dbe84660dbe36aba1ebf0#r1889246753, I was referring to this comment here.